cmake/modules/FindPython/Support.cmake

```diff
@@ -17,7 +17,7 @@ if (NOT DEFINED _${_PYTHON_PREFIX}_REQUIRED_VERSION_MAJOR)
   message (FATAL_ERROR "FindPython: INTERNAL ERROR")
 endif()
 if (_${_PYTHON_PREFIX}_REQUIRED_VERSION_MAJOR EQUAL 3)
-  set(_${_PYTHON_PREFIX}_VERSIONS 3.8 3.7 3.6 3.5 3.4 3.3 3.2 3.1 3.0)
+  set(_${_PYTHON_PREFIX}_VERSIONS 3.9 3.8 3.7 3.6 3.5 3.4 3.3 3.2 3.1 3.0)
```

Comment:
@YanfeiXu this file was copied from cmake upstream to detect the latest python3 releases. and i want to avoid diverging from the upstream with minimal editing. i just created #38018 to sync this file with cmake 3.18.4. could you take a look at it instead?

Reply:
seems the `FindPython/Support.cmake` uses `list (TRANSFORM ...)` which is missing in older version of cmake. i will drop the change to bump up the `FindPython*` from my PR.

Reply:
Thanks for following up it.